ledger/apply/application_test.go

```diff
@@ -1096,20 +1099,42 @@ func TestAppCallApplyDelete(t *testing.T) {
 	a.Equal(cbr, br)
 	a.Equal(basics.EvalDelta{}, ad.EvalDelta)
 
-	// check deletion on empty balance record - happy case
-	b.pass = true
-	b.balances[sender] = basics.AccountData{}
-	err = ApplicationCall(ac, h, &b, ad, &ep, txnCounter)
-	a.NoError(err)
-	a.Equal(appIdx, b.deAllocatedAppIdx)
-	a.Equal(1, b.put)
-	br = b.balances[creator]
-	a.Equal(cbr, br)
-	br = b.putBalances[creator]
-	a.Equal(basics.AppParams{}, br.AppParams[appIdx])
-	a.Equal(basics.StateSchema{}, br.TotalAppSchema)
-	a.Equal(basics.EvalDelta{}, ad.EvalDelta)
-	a.Equal(uint32(0), br.TotalExtraAppPages)
+	cp = basics.AccountData{
+		AppParams:          map[basics.AppIndex]basics.AppParams{appIdx: params},
+		TotalExtraAppPages: 1,
+	}
+	b.balances[creator] = cp
+
```

Comment:
are lines 1102 - 1107 used? same lines of code seem to be repeated in the for loop below.

Reply:
right, copy-paste leftover I'll remove